syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -443,6 +451,73 @@ def matmul(self, *args, **kwargs):
     __matmul__ = matmul
     mm = matmul
 
+    # Approximations:
+    def exp(self, iterations=8):
+        """Approximates the exponential function using a limit approximation:
+        exp(x) = \lim_{n -> infty} (1 + x / n) ^ n
+
+        Here we compute exp by choosing n = 2 ** d for some large d equal to
+        iterations. We then compute (1 + x / n) once and square `d` times.
+
+        Args:
+            iterations (int): number of iterations for limit approximation
+
+        Ref: https://github.com/LaRiffle/approximate-models
+        """
+        return (1 + self / 2 ** iterations) ** (2 ** iterations)
+
+    def sigmoid(self):
+        """
```

Comment:
Shouldn't we replace the sigmoid approximation by the mathematical definition using the approximated `exp`? Or do you think we lose precision by doing that?

I think we should at least compare the differences between both...

Reply:
Definitely! I'll make a comparison as part of my lib to see what's best, thansk for the suggestion

Reply:
You're welcome!

Maybe also do a time performance comparison, to see what are the trade-offs between both